app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = []
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        people_list.append(Person(name, age))
+    for person in people:
+        name = person["name"]
+        if person.get('wife') is None and person.get('husband') is None:
+            continue
+        elif person.get('wife') is not None:
+            Person.people[name].wife = Person.people[person["wife"]]
+        else:
+            Person.people[name].husband = Person.people[person["husband"]]
```

Comment:
You can rewrite in 4 lines with double check
`if "wife" in human and human["wife"] is not None:
            Person.people[human["name"]].wife = Person.people[human["wife"]]`
And same thing for husband
So you dont need 'if' with continue

Reply:
agree with Maksym, better repeat lines 20-21 for husband and you will not need the first check